v1/lib/core/CompLibrary.js

```diff
@@ -8,10 +8,12 @@
 const MarkdownBlock = require('./MarkdownBlock.js');
 const Container = require('./Container.js');
 const GridBlock = require('./GridBlock.js');
+const Showcase = require('./Showcase.js');
```

Comment:
Don't make it part of the exported component library. The custom styles are not part of the library and users who import the library will see unstyled contents.